src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DoNotDirectlyAwaitATaskTests.Fixer.cs

```diff
@@ -252,7 +253,20 @@ public async Task M()
             {
                 TestState = { Sources = { code } },
                 FixedState = { Sources = { fixedCode } },
-                BatchFixedState = { Sources = { fixAllCode } },
+                BatchFixedState =
+                {
+                    Sources = { fixAllCode },
+                    ExpectedDiagnostics =
```

Comment:
Just wondering if we should open a ticket for this? It seems rather weird to have fixer introducing compiler error. WDYT?

Reply:
It's definitely a bug. It impacts both the C# and the VB fixer.

Reply:
See #3910